### PR TITLE
AEIM-2744 Create a unison role

### PR DESCRIPTION
**Pull request by sulemandiwan**

### CODE REVIEW

**Comment by sulemandiwan** [comment]
Apologies I just added the ignore modification. Should have done a WIP but didn't realize till later that it needed that. :/

File: spec/defines/unison_client_spec.rb [hunk 1]
```diff
@@ -33,24 +33,21 @@
...
+          .with_content(%r{path\s+=\s+path2})
+          .with_content(%r{batch\s+=\s+true})
+          .with_content(%r{confirmbigdel\s+=\s+true})
+          .with_content(%r{prefer\s+=\s+newer})
+          .with_content(%r{group\s+=\s+true})
+          .with_content(%r{logfile\s+=\s+/var/log/unison_myinstance.log})
+          .with_content(%r{owner\s+=\s+true})
+          .with_content(%r{times\s+=\s+true})
+          .with_content(%r{numericids\s+=\s+true})
+          .without_content(%r{ignore\s+=\s+Path})
```
Comment (daaang) [inline spec/defines/unison_client_spec.rb:50]: Haha nice `without` there. Was just looking for that and was not disappointed.

**Comment by sulemandiwan** [comment]
Okay squashed and pushed. Will merge shortly.

**Comment by sulemandiwan** [comment]
We are waiting to merge depending on other things as per the ticket. Just noting this is on hold till we are ready to merge things.

**Comment by sulemandiwan** [comment]
Additional changes made to this after we switch to a separate role.

**Pull request merged**